core/services/service.go

```diff
@@ -41,7 +40,7 @@ type (
 	//  	return f
 	//  }
 	//
-	//  func (f *foo) Start() error {
+	//  func (f *foo) Start(ctx context.Context) error {
```

Comment:
Maybe make a note that the context is intended to be used only for duration of Start and not for lifecycle after the service has started?

Reply:
This is explained next to Start method below. Do you want me to duplicate that comment?